Application.groovy

```diff
@@ -1,7 +1,6 @@
-@Grab("edu.wisc.my.restproxy:json-proxy-service:1.0")
+@Grab("edu.wisc.my.restproxy:json-proxy-service:1.1.6-SNAPSHOT")
```

Comment:
Maybe Travis dropped the ball then, because it couldn't be resolved until I did `mvn install`.  I forgot about this, so thanks for catching it, @nblair .  On the other hand, I hate having artifacts in a repo depend on artifacts from a previous revision of the same repo... it becomes its own grandpa or something.